### PR TITLE
Infoschema,Session : fix bugs of unit tests

### PR DESCRIPTION
**Pull request by Orion7r**
## What problem does this PR solve?

* fix bugs about unit test TestIsolationReadDoNotFilterSystemDB and TestBootstrapWithError
* Restoring unit tests TestStmtSummaryTable

### What's Changed:

 * The SQL statement that creates the system view is changed to CREATE OR REPLACE.
 * Change the system table tables to uppercase.
 * Restoring unit tests TestStmtSummaryTable

### How it Works:

 * unit test TestIsolationReadDoNotFilterSystemDB and TestBootstrapWithError are passed.

### CODE REVIEW

**Review by pupillord — APPROVED** [review]
LGTM

**Comment by RogueJin** [comment]
@hey-hoho please help to check these ut

File: infoschema/tables_test.go [hunk 1]
```diff
@@ -981,9 +981,9 @@ func (s *testTableSuite) TestStmtSummaryTable(c *C) {
 		from information_schema.statements_summary
 		where digest_text like 'select * from t%'`,
 	).Check(testkit.Rows("Select test test.t t:k 1 2 0 0 0 0 0 0 0 0 0 select * from t where a=2 \tid                \ttask     \testRows\toperator info\n" +
-		"\tIndexLookUp_10    \troot     \t10     \t\n" +
-		"\t├─IndexRangeScan_8\tcop[tikv]\t10     \ttable:t, index:k(a), range:[2,2], keep order:false, stats:pseudo\n" +
-		"\t└─TableRowIDScan_9\tcop[tikv]\t10     \ttable:t, keep order:false, stats:pseudo"))
+		 "\tIndexLookUp_10    \troot     \t100     \t\n" +
+	     "\t├─IndexRangeScan_8\tcop[tikv]\t100     \ttable:t, index:k(a), range:[2,2], keep order:false, stats:pseudo\n" +
+	     "\t└─TableRowIDScan_9\tcop[tikv]\t100     \ttable:t, keep order:false, stats:pseudo"))
```
Comment (hey-hoho) [inline infoschema/tables_test.go:986]: There is an extra space in these lines. Please check it.

**Review by hey-hoho — APPROVED** [review]
LGTM

**Pull request merged**